File: examples-sdk/zig/803-trigger-webhook-receiver/main.zig
// 803-trigger-webhook-receiver — Zig SDK equivalent
// Compile: vil compile --from zig --input 803-trigger-webhook-receiver/main.zig --release

const vil = @import("vil");

pub fn main() void {
    var server = vil.Server.init("app", 8080);
    server.compile();
}

File: examples-sdk/go/803-trigger-webhook-receiver/main.go
// 803-trigger-webhook-receiver — Go SDK equivalent
// Compile: vil compile --from go --input 803-trigger-webhook-receiver/main.go --release
package main

import vil "github.com/OceanOS-id/vil-go"

func main() {
	s := vil.NewServer("app", 8080)
	s.Compile()
}
